nf_core/pipeline-template/nextflow.config

```diff
@@ -198,6 +197,9 @@ env {
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+// Set default docker registry (will not be used unless pulling docker images)
+docker.registry = 'quay.io'
```

Comment:
Should we have that for podman too?
Or do we assume people using podman will use podman profile?

Reply:
Done!